examples/jaeger-mixin/README.md

```diff
@@ -0,0 +1,27 @@
+# Prometheus monitoring mixin for Jaeger
+
+A set of customisable alerts and dashboards for Jaeger. 
+
+Instructions for use are the same as the [kubernetes-mixin](https://github.com/kubernetes-monitoring/kubernetes-mixin).
+
+Originally developed by [Grafana Labs](https://github.com/grafana/jsonnet-libs/tree/master/jaeger-mixin).
+
+## Background
+
+* For more information about monitoring mixins, see this [design doc](https://docs.google.com/document/d/1A9xvzwqnFVSOZ5fD3blKODXfsat5fg6ZhnKu9LK3lB4/edit#).
```

Comment:
I'm starting a blog post that will use this, I intend to replace most of this readme once the blog post is ready.